src/NuGet.Core/NuGet.Commands/RestoreCommand/RequestFactory/ProjectJsonRestoreRequestProvider.cs

```diff
@@ -69,7 +69,7 @@ public Task<bool> Supports(string path)
             // Get settings relative to the input file
             var settings = restoreContext.GetSettings(file.DirectoryName);
 
-            var sources = restoreContext.GetEffectiveSources(settings);
+            var sources = restoreContext.GetEffectiveSources(settings, null);
```

Comment:
Named parameters would be helpful here (I know this whole thing needs to go away though)